aiounifi/models/api.py

```diff
@@ -27,6 +29,36 @@ def full_path(self, site: str, is_unifi_os: bool) -> str:
             return f"/proxy/network/api/s/{site}{self.path}"
         return f"/api/s/{site}{self.path}"
 
+    def prepare_data(self, raw: bytes) -> TypedApiResponse:
```

Comment:
```suggestion
    def decode(self, raw: bytes) -> TypedApiResponse:
```

Lets rename this to decode.

I wonder if we should introduce a TypedApiResponseV2 that handles converting the response to a common format. Lets not do this now...

Reply:
Ok, decode is a much better name; propagated this in the rest of the code of course.

Maybe in the future a conversion to a common format would be interesting. I agree that introducing this now, probably would take us too far.